templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -328,6 +311,33 @@ spec:
   - {{ . }}
 {{- end }}
 
+{{- if getenv "NODE_MAX_SIZE_PER_AZ" }}
+{{ range (getenv "KOPS_NODES_AVAILABILITY_ZONES" ( getenv "KOPS_AVAILABILITY_ZONES" ) | strings.Split ",") }}
+---
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  labels:
+    kops.k8s.io/cluster: {{ getenv "KOPS_CLUSTER_NAME" }}
+  name: nodes-{{ . }}
+spec:
+  detailedInstanceMonitoring: {{ getenv "KOPS_CLOUDWATCH_DETAILED_MONITORING" "false" }}
+  associatePublicIp: false
+  {{- if bool (getenv "KOPS_CLUSTER_AUTOSCALER_ENABLED" "false") }}
+  cloudLabels:
+    k8s.io/cluster-autoscaler/enabled: "true"
+    k8s.io/cluster-autoscaler/{{ getenv "KOPS_CLUSTER_NAME" }}: "owned"
+  {{- end }}
+  image: {{ getenv "KOPS_BASE_IMAGE" }}
+  machineType: {{ getenv "NODE_MACHINE_TYPE" }}
+  maxSize: {{ getenv "NODE_MAX_SIZE_PER_AZ" }}
+  minSize: {{ getenv "NODE_MIN_SIZE_PER_AZ" "1" }}
+  role: Node
+  subnets:
+  - {{ . }}
```

Comment:
what does `- {{ . }}` mean for subnets in this context?

Reply:
Yes, earlier in the template we create 1 subnet per AZ and name the subnet with the AZ name.